### PR TITLE
add couchdb source as optional install

### PR DESCRIPTION
**Pull request by lionelvillard**
## Proposed Changes 
 
- add couchdb source as optional install
-

### CODE REVIEW

File: docs/install/knative-with-any-k8s.md [hunk 1]
```diff
@@ -572,7 +572,21 @@ To learn more about the Cloud Audit Logs source, try [our sample](../eventing/sa
 
 {{< /tab >}}
 
-<!-- TODO: couchdb source -->
+{{% tab name="CouchDB Source" %}}
```
Comment (matzew) [inline docs/install/knative-with-any-k8s.md]: Apache CouchDB 

is trademark compliant usage

Comment (lionelvillard) [reply docs/install/knative-with-any-k8s.md]: indeed!

**Comment by lionelvillard** [comment]
@mattmoor any ideas how to fix/disable the link check?

**Pull request merged**